platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -1524,4 +1525,14 @@ export default class HangingProtocolService extends PubSubService {
   _copyProtocol(protocol: Protocol) {
     return JSON.parse(JSON.stringify(protocol));
   }
+
+  /** Register the hanging protocols from the extension module */
+  public initModule(extensionModule, extensionId: string): void {
```

Comment:
extensonModule => hangingProtocolModule
remove extensionId - it is still there, but unused